src/vaults/vaults.utils.ts

```diff
@@ -209,16 +209,39 @@ export async function getVaultPerformance(
   let vaultSources: CachedValueSource[] = [];
   try {
     vaultSources = await loadVaultEventPerformances(chain, vaultDefinition);
-    vaultSources.push();
   } catch (err) {
     if (DEBUG) {
       console.log(`${vaultDefinition.name} vault APR estimation fallback to badger subgraph`);
     }
     vaultSources = await loadVaultGraphPerformances(chain, vaultDefinition);
   }
+  const vaultApr = vaultSources.reduce((total, s) => total + s.apr, 0);
+  // if we are not able to measure any on chain, or graph based increases falleback to ppfs measurement
+  if (vaultApr === 0) {
```

Comment:
this deals with vaults like the yearn vault that we have no info on - it measures our saved snapshots to make estimates based off what data we can if possible